oio/xcute/common/backend.py

```diff
@@ -641,6 +669,21 @@ def get_job_info(self, job_id):
     def _get_job_info(self, job_id, client):
         return client.hgetall(self.key_job_info % job_id)
 
+    @handle_redis_exceptions
+    def list_locks(self):
+        locks = self.conn.hgetall(self.key_locks)
+
+        return [
+            dict(lock=lock[0], job_id=lock[1])
+            for lock in sorted(locks.items())
+        ]
```

Comment:
Why return a list of dictionaries `[{"lock": LOCK, "job_id": JOB_ID}, ...]` instead of the dictionary `{LOCK: JOB_ID, ...}`?
Also, if the client wants to sort the locks, it can do it itself. This will allow a faster response and free up the server.